drizztsaga/scripts/snip/eRestorWeapon.baf

```diff
@@ -0,0 +1,54 @@
+
+IF
+	!InParty("F_DRIZZT")  // Drizzt
+	Dead("F_DRIZZT")  // Drizzt
+	HasItem("F_DRIWP1",Player1) // Icingdeath +3
+THEN
+	RESPONSE #100
+		TakeItemReplace("sw1h15","F_DRIWP1",Player1)
```

Comment:
what about the second Drizzt's weapon `F_DRIWP2`?

Reply:
`F_DRIWP2` is undroppable so nothing to worry.